test/run.sh

```diff
@@ -10,24 +10,26 @@ RF_VENV="${ROOTDIR}/_output/robotenv"
 RF_VARIABLES="${SCRIPTDIR}/variables.yaml"
 DRYRUN=false
 OUTDIR="${ROOTDIR}/_output/e2e-$(date +%Y%m%d-%H%M%S)"
-
+RAMDOMIZE="all"
+STRESS_TESTING_OPTS=""
 
 function usage {
     local -r script_name=$(basename "$0")
     cat - <<EOF
-${script_name} [-h] [-n] [-o output_dir] [-v venv_dir] [-i var_file] [test suite files]
+${script_name} [-h] [-n] [-o output_dir] [-v venv_dir] [-i var_file] [-s name-value] [test suite files]
 
 Options:
 
-  -h       Print this help text.
-  -n       Dry-run, do not run the tests.
-  -o DIR   The output directory. (${OUTDIR})
-  -v DIR   The venv directory. (${RF_VENV})
-  -i PATH  The variables file. (${RF_VARIABLES})
+  -h                 Print this help text.
+  -n                 Dry-run, do not run the tests.
+  -o DIR             The output directory. (${OUTDIR})
+  -v DIR             The venv directory. (${RF_VENV})
+  -i PATH            The variables file. (${RF_VARIABLES})
+  -s NAME-VALUE      To enable an stress scenario.
```

Comment:
This is a generic script for running any suites. I do not think we should add options that are specific to stress-testing here. Instead, we just need to document how to run it with all the options.

Reply:
My idea is to add an opt-in option to enable and disable any Stress condition before running any test.